### PR TITLE
sbom-tool 1.5.1

### PR DESCRIPTION
**Pull request by chenrui333**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

**Review by ZhongRuoyu — COMMENTED** [review]
Not marked as the latest release yet: https://github.com/microsoft/sbom-tool/releases

File: Formula/s/sbom-tool.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class SbomTool < Formula
   desc "Scalable and enterprise ready tool to create SBOMs for any variety of artifacts"
   homepage "https://github.com/microsoft/sbom-tool"
-  url "https://github.com/microsoft/sbom-tool/archive/refs/tags/v1.5.0.tar.gz"
-  sha256 "6997e195ce2cb0902d913ec23cec0ebb015f90d1ab26c861beaaaadf4f456391"
+  url "https://github.com/microsoft/sbom-tool/archive/refs/tags/v1.5.1.tar.gz"
+  sha256 "ebc52490acbe7509f880c93737b5051202a2173215ddb975550f61ea3f3b1a2d"
   license "MIT"
   head "https://github.com/microsoft/sbom-tool.git", branch: "main"
```
Comment (SMillerDev) [inline Formula/s/sbom-tool.rb:8]: Needs a livecheck

**Review by SMillerDev — DISMISSED** [review]

File: Formula/s/sbom-tool.rb (hunk 1 above)
Comment (chenrui333) [reply Formula/s/sbom-tool.rb:8]: livecheck :alert: :D

**Comment by chenrui333** [comment]
logged an upstream issue about the release status, microsoft/sbom-tool#321

**Comment by chenrui333** [comment]
now it is the latest release, and the checksum stays the same

**Review by ZhongRuoyu — APPROVED** [review]

**Review by p-linnane — APPROVED** [review]

**Pull request merged**